mcumgr-core/src/test/java/io/runtime/mcumgr/McuMgrImageTest.kt

```diff
@@ -0,0 +1,35 @@
+package io.runtime.mcumgr
+
+import io.runtime.mcumgr.image.McuMgrImage
+import org.junit.Test
+import java.io.ByteArrayOutputStream
+import java.io.InputStream
+
+class McuMgrImageTest {
+
+    @Test
+    fun `parse image without protected tlvs success`() {
+        val inputStream = this::class.java.classLoader?.getResourceAsStream("slinky-no-prot-tlv.img")
```

Comment:
In a unit test it's appropriate to use `!!` to keep code short and to the point.

```suggestion
        val inputStream = this::class.java.classLoader?.getResourceAsStream("slinky-no-prot-tlv.img")!!
```